packages/documentation/src/layouts/layout.js

```diff
@@ -154,7 +162,7 @@ class Layout extends React.Component {
             siteTitle={data.site.siteMetadata.title}
             allMarkDown={data.allMarkDown.edges}/>'
 
-            '<div className="ContentArea">{children}</div>'
+            '<div className="ContentArea docSearch-content">{children}</div>'
```

Comment:
![screen shot 2019-03-06 at 08 15 18](https://user-images.githubusercontent.com/4998130/53891611-0035d000-3fe8-11e9-8c5a-6b4939cda7f0.png)

Reply:
This will be used by the config to determine organize the content by levels. I'm not exactly sure how to follow this recommendation: 
![screen shot 2019-03-06 at 08 22 09](https://user-images.githubusercontent.com/4998130/53892131-f496d900-3fe8-11e9-86e3-17ce89a2e408.png)

In the example config they rely on less specific queries: 

![screen shot 2019-03-06 at 08 20 47](https://user-images.githubusercontent.com/4998130/53892196-12643e00-3fe9-11e9-9640-96ed9f186c87.png)

*They* create the initial config for us so it'll be interesting to see what they do. I'm sure we'll have some back and forth and if they want unique ids on things we can come up with a convention / helper.